components/ui/__tests__/alert.test.tsx

```diff
@@ -45,5 +52,12 @@ describe("Alert", () => {
       "border-red-400",
     );
     expect(container.firstChild?.firstChild).toHaveClass("p-4", "babayaga");
+    expect(
+      container.firstChild?.firstChild?.firstChild?.firstChild?.firstChild,
```

Comment:
Please [query the component that you're targeting properly](https://testing-library.com/docs/queries/about).